RollOfHonour/src/RollOfHonour.Core/Models/Person.cs

```diff
@@ -1,17 +1,88 @@
-﻿namespace RollOfHonour.Core.Models;
+﻿using System.Collections.ObjectModel;
+
+namespace RollOfHonour.Core.Models;
 
 public class Person
 {
   public int Id { get; set; }
   public string FirstNames { get; set; } = string.Empty;
   public string Initials { get; set; } = string.Empty;
   public string LastName { get; set; } = string.Empty;
-  public string Rank { get; set; } = string.Empty;//TODO: This needs cleaning up - so many similar entries. Autopicker/Prompt perhaps
+
+  public string Rank { get; set; } =
+    string.Empty; //TODO: This needs cleaning up - so many similar entries. Autopicker/Prompt perhaps
```

Comment:
We'll definitely need to standardise this list, yes.

Reply:
Formatted code. XD